step.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# 'read' has to be before 'set -e'
+read -r -d '' UNAVAILABLE_MESSAGE << EOF_MSG
+Bitrise Build Cache is not activated in this build.
+
+You have added the **Activate Bitrise Build Cache for Bazel** add-on step to your workflow.
+
+However, you don't have an activate Bitrise Build Cache Trial or Subscription for the current workspace yet.
+
+You can activate a Trial at [app.bitrise.io/build-cache](https://app.bitrise.io/build-cache),
+or contact us at [support@bitrise.io](mailto:support@bitrise.io) to activate it.
+EOF_MSG
+
+set -eo pipefail
+
+echo "Checking whether Bitrise Build Cache is activated for this workspace ..."
+if [ "$BITRISEIO_BUILD_CACHE_ENABLED" != "true" ]; then
```

Comment:
I am not familiar with this flow, how is this variable being populated?

Reply:
It is set here: https://github.com/bitrise-io/bitrise-website/blob/b2c588938831a7a35f626b80a72e5e119b2bde76/components/ci/app/helpers/workflow_helper.rb#L244

Reply:
Thanks!